README.md

```diff
@@ -1,2 +1,24 @@
 # bedrock-karma
 Karma test framework for bedrock modules
+
+## Quick Examples
+
+```
+npm install bedrock-karma
+```
+
+```js
+const {config} = bedrock;
+
+// add karma tests for your module/project to your config file
+config.karma.suites['bedrock-foo'] = '/foo/test/tests/**/*.js';
+```
+
+
+## Configuration
+
+For more documentation on configuration, see [config.js](./lib/config.js).
+
+
+[bedrock]: https://github.com/digitalbazaar/bedrock
+[karma]: https://github.com/karma-runner/karma
```

Comment:
Did you mean to use these links in the description?

Reply:
Yes, I meant to link these

Reply:
Another way is to leave refs like before and add links to the description at the top: `[Karma][] test framework for [bedrock][] modules.  If left in a references section, maybe make it a bulleted list.